valusets/ValueSet-England-VaccinationProcedure.xml

```diff
@@ -0,0 +1,1907 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-VaccinationProcedure" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-VaccinationProcedure" />
+    <version value="1.0.0" />
+    <name value="EnglandVaccinationProcedure" />
+    <title value="Interim version of UK Core Vaccination Procedure."/>
```

Comment:
Should this have Interim version?

Reply:
Content is almost same with UKCoreVaccinationProcedure. Thus, we might change the mapping to UKCoreVaccinationProcedure